.github/workflows/build.yml

```diff
@@ -2,7 +2,9 @@ name: build
 on: push
 
 env:
-  IMAGE_NAME: vvgo
+  VVGO_IMAGE_NAME: vvgo
+  PAGE_CACHE_IMAGE_NAME: page-cache
```

Comment:
At this point, we might want to look at a full ingress: Traefik, Caddy, etc.

Reply:
Why do we need Traefik and Caddy? What problems will they solve?

Reply:
If need them, we can add them later. I'd like to get this pr merged.